packages/core/RNSentryAndroidTester/app/src/test/java/io/sentry/react/RNSentrySDKTest.kt

```diff
@@ -0,0 +1,138 @@
+package io.sentry.react
+
+import android.content.Context
+import com.facebook.react.bridge.JavaOnlyMap
+import com.facebook.react.bridge.ReadableMap
+import io.sentry.ILogger
+import io.sentry.Sentry.OptionsConfiguration
+import io.sentry.SentryLevel
+import io.sentry.android.core.SentryAndroidOptions
+import org.json.JSONObject
+import org.junit.After
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertThrows
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+import org.mockito.ArgumentMatchers.contains
+import org.mockito.ArgumentMatchers.eq
+import org.mockito.MockedStatic
+import org.mockito.Mockito.mock
+import org.mockito.Mockito.mockStatic
+import org.mockito.Mockito.verify
+import org.mockito.MockitoAnnotations
+
+@RunWith(JUnit4::class)
+class RNSentrySDKTest {
+    private val configurationFile = "sentry.options.json"
+
+    private lateinit var mockLogger: ILogger
+    private lateinit var mockContext: Context
+    private lateinit var mockConfiguration: OptionsConfiguration<SentryAndroidOptions>
+    private lateinit var mockedRNSentryStart: MockedStatic<RNSentryStart>
+    private lateinit var mockedRNSentryJsonUtils: MockedStatic<RNSentryJsonUtils>
+
+    @Before
+    fun setUp() {
+        MockitoAnnotations.openMocks(this)
+        mockLogger = mock(ILogger::class.java)
+        mockContext = mock(Context::class.java)
+        mockConfiguration = mock(OptionsConfiguration::class.java) as OptionsConfiguration<SentryAndroidOptions>
+        mockedRNSentryStart = mockStatic(RNSentryStart::class.java)
+        mockedRNSentryJsonUtils = mockStatic(RNSentryJsonUtils::class.java)
+    }
+
+    @After
+    fun tearDown() {
+        mockedRNSentryStart.close()
+        mockedRNSentryJsonUtils.close()
+    }
+
```

Comment:
Can you also add integration test which won't use the mocks (maybe just to supply the file) and initializes the SDK?